packages/bs-neo3/src/BSNeo3.ts

```diff
@@ -156,6 +160,13 @@ export class BSNeo3<BSName extends string = string>
     this.nftDataService = new GhostMarketNDSNeo3(network)
     this.explorerService = new DoraESNeo3(network)
 
+    this.voteService = new DoraVSNeo3<BSName>({
+      network,
+      feeToken: this.feeToken,
+      validateAddress: this.validateAddress.bind(this),
+      generateSigningCallback: this.generateSigningCallback.bind(this),
+    })
```

Comment:
Instead of passing these properties as parameters, pass the entire BSNeo3
```suggestion
    this.voteService = new DoraVSNeo3<BSName>(this)
```